### PR TITLE
Add a new shape category

### PR DESCRIPTION
**Pull request by aldinaufal21**

### CODE REVIEW

File: qual_id/categories/shapes.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from ..category import Category
+
+class Shapes(Category):
```
Comment (gabrielbarker) [inline qual_id/categories/shapes.py:3]:
```suggestion
class Shape(Category):
```

File: qual_id/pattern.py [hunk 2]
```diff
@@ -3,6 +3,7 @@
 from .categories.food import Food
 from .categories.drink import Drink
 from .categories.color import Color
+from .categories.shapes import Shapes
```
Comment (gabrielbarker) [inline qual_id/pattern.py:6]:
```suggestion
from .categories.shape import Shape
```

File: qual_id/pattern.py [hunk 3]
```diff
@@ -15,6 +16,7 @@ class Pattern:
       "food": Food(),
       "drink": Drink(),
       "color": Color(),
+      "shapes": Shapes(),
```
Comment (gabrielbarker) [inline qual_id/pattern.py:19]:
```suggestion
      "shape": Shape(),
```

File: test/categories/test_shapes.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import unittest
+from qual_id.categories.shapes import Shapes
```
Comment (gabrielbarker) [inline test/categories/test_shapes.py:2]:
```suggestion
from qual_id.categories.shape import Shape
```

File: test/categories/test_shapes.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+import random
+
+
+class TestFood(unittest.TestCase):
```
Comment (gabrielbarker) [inline test/categories/test_shapes.py:6]:
```suggestion
class TestShape(unittest.TestCase):
```

File: test/categories/test_shapes.py [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+  def setUp(self):
+    self.shapes = Shapes()
```
Comment (gabrielbarker) [inline test/categories/test_shapes.py:8]:
```suggestion
    self.shape = Shape()
```

File: test/categories/test_shapes.py [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+  def test__get_values__returns_list(self):
```
Comment (gabrielbarker) [inline test/categories/test_shapes.py:10]: The way that unit tests are written has been simplified. If you pull the latest changes and take a look at another unit test

**Review by gabrielbarker — CHANGES_REQUESTED** [review]
Hey @aldinaufal21 I've spotted some things that need changing. The category name should be 'Shape', not 'Shapes', so please can you _change the file names_ as well as the suggestions I've made
Thanks!

File: test/categories/test_shapes.py (hunk 7 above)
Comment (gabrielbarker) [reply test/categories/test_shapes.py]: This hasn't been resolved.

**Review by gabrielbarker — CHANGES_REQUESTED** [review]
You still need to pull the latest changes and change the way the test works

File: test/categories/test_shape.py [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+from qual_id.categories.shape import Shape
+import random
+
+
+class TestShape(unittest.TestCase):
+  def setUp(self):
+    self.shape = Shape()
+
+  def test__get_values__is_valid(self):
+        self.assertTrue(CategoryHelper.get_values_is_valid(self.shape))
```
Comment (gabrielbarker) [inline test/categories/test_shape.py:11]: As you can see from the error message in the tests, you need to import `CategoryHelper`, just like you did [here](https://github.com/gabrielbarker/qual-id/blob/main/test/categories/test_color.py)
Also you don't need `import random`

**Review by gabrielbarker — CHANGES_REQUESTED** [review]

File: test/categories/test_shape.py (hunk 8 above)
Comment (aldinaufal21) [reply test/categories/test_shape.py:11]: sorry, i will fix it

**Review by gabrielbarker — APPROVED** [review]
Perfect! Thanks @aldinaufal21

**Pull request merged**